app/controllers/api/reviews_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::ReviewsController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: [:create]
+
+  def create
+    review = Review.new(
+      truck_id: params[:truck_id],
```

Comment:
your API should send a `review` json object so that you have a `params[:review]`